server/lib/configuration.js

```diff
@@ -19,7 +19,8 @@ const conf = module.exports = convict({
   allowed_parent_origins: {
     default: [],
     doc: 'Origins that are allowed to embed FxA within an IFRAME',
-    format: Array
+    format: Array,
+    env: 'ALLOWED_PARENT_ORIGINS'
```

Comment:
Seems the linting rules dislike that the property names are not in alphabetical order, such as `env` should be before `format`. All the way down.